src/Compilers/Core/CodeAnalysisTest/ShadowCopyAnalyzerAssemblyLoaderTests.cs

```diff
@@ -1,146 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
These tests were merged into `DefaultAnalyzerAssemblyLoaderTestsBase`. That way they run on the four major configurations that we care about.